samtools.1

```diff
@@ -1536,15 +1536,43 @@ The three categories used are:
 
 0 : Either both READ1 and READ2 are set; or neither is set.
 
+The exact meaning of these categories depends on the sequencing technology
+used.
```

Comment:
I guess it depends also on the mapper (in particular the cases that are caught with -0)

Reply:
This text actually came from the [SAM](http://samtools.github.io/hts-specs/SAMv1.pdf) specification.  These bits are supposed to show the relative locations of the reads on the piece of DNA that was sequenced.  They're set by the sequencing protocol in use, and are an input to the mapper rather than an output (or should be for mappers that understand SAM/BAM/CRAM files as input).

Reply:
OK. Thanks for the info. Didn't know that. Since I tell most mappers which files contain read1 and a which contains read2 I thought the mapper generates the info.